blueocean-bitbucket-pipeline/pom.xml

```diff
@@ -43,18 +43,14 @@
     <dependency>
       <groupId>commons-codec</groupId>
       <artifactId>commons-codec</artifactId>
+      <scope>provided</scope>
```

Comment:
do not bundle this

Reply:
maybe just remove the `<dependency>` as it's already declared in `<dependencies>` section in parent pom . a bit redundant?

Reply:
possibly but the other day you where saying if you use it declare it :)

this is known to work - I say ship it.